packages/widgets/resources/views/chart-widget.blade.php

```diff
@@ -23,17 +23,28 @@
                 </x-filament::card.header>
 
                 @if ($filters)
-                    <select
-                        class="text-gray-900 border-gray-300 block h-10 transition duration-75 rounded-lg shadow-sm focus:border-primary-500 focus:ring-1 focus:ring-inset focus:ring-primary-500 dark:bg-gray-700 dark:border-gray-600 dark:text-gray-200 dark:focus:border-primary-500"
-                        wire:model="filter"
-                        wire:loading.class="animate-pulse"
-                    >
-                        @foreach ($filters as $value => $label)
-                            <option value="{{ $value }}">
-                                {{ $label }}
-                            </option>
-                        @endforeach
-                    </select>
+                    <div class="flex items-center gap-3">
+                        @if ($showFilterLoadingIndicator)
+                            <x-filament::loading-indicator
+                                x-cloak="x-cloak"
+                                class="h-8 w-8 text-gray-500"
```

Comment:
These size values look odd to me? (Haven't tested visually, but just from the code.)

Reply:
Also not sure about the color

Reply:
Here's what it looks like: 
<img width="803" alt="image" src="https://user-images.githubusercontent.com/4503765/215603051-ffd4561c-2380-4a5c-a389-f94d17bb227b.png">

Reply:
![2023-01-30_15-48-25 (1)](https://user-images.githubusercontent.com/4503765/215603179-c1e2c926-9005-4d9a-b1a4-c4ef890b0d3b.gif)